src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsregisterstodeliusupdate/helpers/Mesages.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.hmppsregisterstodeliusupdate.helpers
+
+fun courtRegisterUpdateMessage() = """
+      {
+        "Type": "Notification", 
```

Comment:
out of interest - why have these as kotlin files instead of json files in resources?

Reply:
I find them just as easy to read, and easier to find as json files.

Also gives  a bit more flexibility when we start parameterising them, change the court id for instance